### ISSUE TITLE
Add hotkey to cycle tabs in main window (CTRL+TAB on Windows)

### ISSUE THREAD

**Issue by gnbl** [sqlitebrowser/sqlitebrowser#2397]
While there are [hotkeys to select a specific tab](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Keyboard-shortcuts#main-window), I'd like it if I could switch between tabs via hotkey, by order of "most recently used". 
CTRL+TAB is often used for this on Windows applications (https://en.wikipedia.org/wiki/Table_of_keyboard_shortcuts#Tab_management).

**Comment by lucydodo** [comment]
It seems to be a reasonable idea. 😄 However, currently the `Ctrl + Tab` shortcut is used to move between the main tabs.
Do you have any other shortcuts you would recommend?

**Comment by gnbl** [comment]
That is almost exactly what I would like, however:
1. I did not notice something was implemented as it does not trigger in _Browse Data_ (moves to next cell) and  _Execute SQL_ (nothing)
2- it's not documented.

So it appears to be a bug. I'm on "DB Browser for SQLite Version 3.12.99 (Sep 12 2020)" (no commit hash in About dialog).
When that is fixed, it would probably require the addition of a list of tabs that is modified when a tab is activated (release both keys).

**Comment by lucydodo** [comment]
Sorry. Looks like I'm wrong. 😞 
Are you saying that it would be better to move the tabs in the order last used,
rather than using the 'Ctrl + Tab' shortcut to move tabs in order(the currently implemented method)?

**Comment by gnbl** [comment]
Yes, that would be my preference. That feature still allows changing to any tab, but makes switching back and forth between two tabs much nicer.
In addition, I am saying that the current implementation to change tabs with ctrl+tab does not work for me in half the tabs.
Alt+1/2/3/4 work, but CTRL+TAB only works for 1->2 and 3->4. Browse Data and Execute SQL either don't register the hotkey, or they do not propagate the event.

**Comment by lucydodo** [comment]
We can give the option to change `Ctrl + Tab`'s mechanism of operation according to user's preferences,
but I think we need to hear from others. What do you reckon?, @MKleusberg 

Also, I'm just installing Windows on the VM to test for what seems to be a bug you mentioned.

**Comment by lucydodo** [comment]
Well, I've tested it, but I'm working fine.
Would you like to try it on the latest [nightly build](https://nightlies.sqlitebrowser.org/latest/)?

**Comment by gnbl** [comment]
The only occurance of "Ctrl+Tab" seems to be in https://github.com/sqlitebrowser/sqlitebrowser/blob/master/src/MainWindow.cpp#L196
I have set US English [default] as UI language.

Not sure if this could be related: https://github.com/sqlitebrowser/sqlitebrowser/blob/master/src/sqlitetablemodel.cpp#L407
Several places act on Qt::ControlModifier.

I'll try the latest nightly in 12+ hours.

**Comment by gnbl** [comment]
https://nightlies.sqlitebrowser.org/latest/DB.Browser.for.SQLite-win64.zip 14-Sep-2020 00:34            20268257

- works if no database is opened 
- stops working on Browse Data and Execute SQL when tabs get focus (click into empty Database Structure or Execute SQL tab)
- with database open
  - works in Browse Data if table is empty, even if empty table view clicked
  - if "Browse Data" has table with data and was focused once, CTRL+TAB moves to next cell

Quick test shows 32 bit version is also quirky about this.

**Comment by lucydodo** [comment]
Thanks for confirm this :smile: 

>if "Browse Data" has table with data and was focused once, CTRL+TAB moves to next cell

This seems to be a bug. We specified 'Tab' to move to the next cell in 'Browse Data', but there seems to be something wrong.

>stops working on Execute SQL

This is because in the 'Execute SQL' tab, 'Ctrl + Tab' is overridden to use to move the tabs of the SQL editor.
Maybe we have to think about other shortcuts we can use in 'Execute SQL' tab. :thinking:
